Dining/models.py

```diff
@@ -40,6 +40,13 @@ class DiningList(models.Model):
     The following fields may not be changed after creation: kitchen_cost, min_diners/max_diners!
     """
     date = models.DateField()
+
+    """Todo: the date+association combination determines the URL. This makes it impossible to have multiple dining lists
+    of the same association on the same day. Probably need to change that"""
```

Comment:
That's not what's meant here, I agree with not allowing associations to claim more than 1 slot per day. This note is therefore also not important, but I meant that the URL layout shouldn't be enforcing that restriction, it should be enforced by validation checks in the code (only), so that it's possible to make exemptions or change rules later if needed. That will also make the url layout simpler.

Reply:
We can indeed simpliy it, but personally I like some readability in the url. In the same way that we have e.g. /associations/Knights/members instead of /a/2/m. That way we can share urls in chats and without clicking you could know where links lead to which is imy always a plus.